src/pretix/plugins/banktransfer/payment.py

```diff
@@ -118,6 +118,16 @@ def test_mode_message(self):
     def settings_form_fields(self):
         d = OrderedDict(
             list(super().settings_form_fields.items()) + list(BankTransfer.form_fields().items()) + [
+                ('code_prefix', forms.CharField(
+                    label=_('Bank transfer code prefix.'),
+                    help_text=_('This value is prepended to the unique order code.'),
```

Comment:
We should add something like "If not set, the short form of the event name will be used."